### PR TITLE
Feature/expose repo path

### PR DESCRIPTION
**Pull request by thespain**
@genebean, Please see the commits in this PR for the issue #6 I submitted. I have this working in a development environment using an alternate `repo_path` to the `/deploy` webhook.

The Docker image I pushed based on my fork is located at https://hub.docker.com/r/thespain/genebean-treeplanter/

Please set me know if there's any changes you'd like to see.

Thanks!

### CODE REVIEW

**Comment by genebean** [comment]
Looking great! Are you up to adding tests to [test/travisci_test_running_container.sh](test/travisci_test_running_container.sh) to verify it works? Also, add the phrase `fixes #6` to your commit message.

**Comment by thespain** [comment]
Absolutely, I'll take care of it! Do you prefer all changes squashed into one commit or leave as 2 or 3?

**Comment by genebean** [comment]
Two seems fitting from what I see above but your call.

**Comment by thespain** [comment]
Just finished adding tests for the new functionality and `fixes #6` to the commit message. I tried to follow your formatting in the testing script, but just let me know if you want to see any changes.

Thanks!

File: docker.pp [hunk 1]
```diff
@@ -20,18 +20,18 @@
 
 # this is where your git repo(s) will live
 file { "/home/${appuser}/trees":
-  ensure   => 'directory',
-  group    => $appuser,    # generally the same as your app user
-  mode     => '755',       # adjust as needed
-  owner    => $appuser,    # must be your app user
+  ensure => 'directory',
+  group  => $appuser,    # generally the same as your app user
+  mode   => '0755',       # adjust as needed
```
Comment (genebean) [inline docker.pp]: Please line up comments

**Comment by thespain** [comment]
Done

**Comment by genebean** [comment]
I'll do a release tonight to get this into master.

**Comment by genebean** [comment]
@thespain - Here in just a little bit Docker will automatically build a new image based on the development branch. Will you test it out before I do a full release please? It has your stuff from here plus #8's stuff in it.

**Comment by genebean** [comment]
Running `docker pull genebean/tree-planter:develop` should get the new one that built a few minutes ago.

**Pull request merged**